DTS-STN/canadian-dental-care-plan
# Reorganizing cache duration environment variables

### Description
* Added missing cache duration environment variables as some were being reused
* Cache duration variables sorted in alphabetical order and names cleaned up
* Cache duration variable now accepts seconds instead of milliseconds

### Related Azure Boards Work Items
In preparation for [AB#3105](https://dev.azure.com/DTS-STN/Canada%20Dental%20Care%20Plan/_workitems/edit/3105)

### Checklist

### Test Instructions
Run the application and ensure that the various services used to retrieve lookup values isn't broken.
